scripts/dockerfile_fragments.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+from __future__ import annotations
```

Comment:
Could you add a section for this script on the folders readme.md?

Reply:
@coderabbitai done